bindata/etcd/pod.yaml

```diff
@@ -300,6 +288,38 @@ ${COMPUTED_ENV_VARS}
       requests:
         memory: 70Mi
         cpu: 50m
+  - name: etcd-readyz
+    image: ${OPERATOR_IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command: [ "cluster-etcd-operator", "readyz" ]
+    args:
+      - --target=https://localhost:2379
+      - --listen-port=7443
+      - --cert-file=$(ETCDCTL_CERT)
+      - --key-file=$(ETCDCTL_KEY)
+      - --cacert-file=$(ETCDCTL_CACERT)
+    ports:
+    - containerPort: 7443
+    resources:
+      requests:
+        memory: 50Mi
+        cpu: 10m
+    env:
+${COMPUTED_ENV_VARS}
+      - name: "ETCD_STATIC_POD_VERSION"
+        value: "REVISION"
+      - name: "OPENSHIFT_PROFILE"
```

Comment:
this looks a little odd, what is it for?

Reply:
Carryover from the `etcd` container template. Removed as we don't really need these.